packages/react-devtools-shell/src/app/index.js

```diff
@@ -32,6 +32,7 @@ ignoreErrors([
   'Warning: %s is deprecated in StrictMode.', // findDOMNode
 ]);
 ignoreWarnings(['Warning: componentWillReceiveProps has been renamed']);
+ignoreLogs([]);
```

Comment:
Devtools tests compatibility with legacy APIs, and this set of ignore messages suppresses their error messages in the testing shell because they might mask more serious error messages. There aren't any `console.log` calls that need to be ignored, but I added it here for consistency (since the devtools shell test harness has its own console override).